static/js/popovers.js

```diff
@@ -970,14 +970,20 @@ export function register_click_handlers() {
         e.preventDefault();
     });
 
-    $("body").on("click", ".update_status_text", (e) => {
+    function open_user_status_modal(e) {
         hide_all();
 
         user_status_ui.open_user_status_modal();
 
         e.stopPropagation();
         e.preventDefault();
-    });
+    }
+
+    $("body").on("click", ".update_status_text", open_user_status_modal);
+
+    // The first item in the buddy list is one's own.
+    // Clicking on one's own status emoji should open the user status modal.
+    $("#user_presences").on("click", "li:first-child .status_emoji", open_user_status_modal);
```

Comment:
We should avoid using `first-child` selectors; they end up breaking when the context changes (E.g. if you're doing a search in the sidebar that doesn't include yourself).  Just add a `user_sidebar_entry_me` class in the sidebar for the current user and key off that.

Reply:
Got it.